protocols/Imperial College/README.md

```diff
@@ -0,0 +1,38 @@
+# Drug Mixing and Transferring
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+### Partner
+[Imperial College]
```

Comment:
No `[]` brackets here - you only need those for links in markdown (it will show up with the brackets on the webpage, which looks weird: "[Imperial College]"